tdrs-backend/tdpservice/reports/urls.py

```diff
@@ -2,11 +2,24 @@
 
 from rest_framework.routers import DefaultRouter
 from . import views
+from django.urls import path
 
 router = DefaultRouter()
 
 router.register("", views.ReportFileViewSet)
 
```

Comment:
depending on if you are an ofa admin or not, this api has different behavior, so we have a route where no stt is passed for data preppers associated with a specific STT, since they are only expected to interact with their own stt, and one with the stt in it for the admin since they do not have a meaningful STT association.